spec/unit/vim_plug_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'exogenesis/passengers/vim_plug'
+
+describe VimPlug do
+  let(:config) { double }
+  let(:executor) { executor_double }
+
+  subject { VimPlug.new(config, executor) }
+
+  describe :up do
+    it 'should install and update plugins' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.